posthog/models/filters/mixins/funnel.py

```diff
@@ -89,7 +89,7 @@ def funnel_order_type(self) -> Optional[FunnelOrderType]:
     def funnel_viz_type(self) -> Optional[FunnelVizType]:
         funnel_viz_type = self._data.get(FUNNEL_VIZ_TYPE)
         if (
-            not funnel_viz_type is None
+            funnel_viz_type is None
```

Comment:
@Twixes wasn't sure if the `not` was intentional?

I'd expect, since `viz_type` is new setting, if it's set, should override `TRENDS_LINEAR`? (and when not set, we wouldn't return `TRENDS`?

Reply:
Ahh, this would explain https://github.com/PostHog/posthog/pull/5015#discussion_r664904315. Looks like I accidentally did `not funnel_viz_type` and `funnel_viz_type is None` _at once_…

Reply:
Ah, I see it now, that makes sense 😂

Reply:
Added a test for this, too. I should've done this when I introduced the change in the first place :$